src/rules/selector-type-no-unknown/README.md

```diff
@@ -0,0 +1,89 @@
+# selector-type-no-unknown
+
+Disallow unknown type selectors.
+
+```css
+       input {}
+/**    ↑
+ * This type selector */
+```
+
+The following patterns are considered warnings:
+
+```css
+unknown { }
+```
+
+```css
+uNkNoWn { }
+```
+
+```css
+UNKNOWN { }
+```
+
+The following patterns are *not* considered warnings:
+
+```css
+input { }
+```
+
+```css
+iNpUt { }
+```
+
+```css
+INPUT { }
```

Comment:
In this and other docs, I think we probably don't want to devote so much space to demonstrate case insensitivity. We could just add a sentence that it's case insensitive. But with this much space devoted to it, it seems like a major feature of the rule, one of the most important, when it's really not.